src/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
+// eslint-disable-next-line import/extensions
+import Books from './redux/books/Books';
+// eslint-disable-next-line import/extensions
+import Categories from './redux/categories/Categories.js';
+
```

Comment:
These two imports raise errors and stop the server to run. Please fix the errors in the dev server.